releasenotes/notes/remove-clone-context-3cc6f3d73f15345e.yaml

```diff
@@ -0,0 +1,4 @@
+---
+upgrade:
+  - |
+    :py:meth:`ddrace.context.Context.clone` was removed. This is no longer need since ddtrace will now work out of the box with async frameworks.
```

Comment:
```suggestion
    :py:meth:`ddrace.context.Context.clone` was removed. This is no longer needed since the tracer now supports async frameworks out of the box.
```